docs/kb/troubleshoot-functions.md

```diff
@@ -5,178 +5,109 @@ title: VMware Event Broker Function Troubleshooting
+kube-system          antrea-controller-85c944dc84-jc28b                1/1     Running     0          14h
+kube-system          coredns-74ff55c5b-rqm7c                           1/1     Running     0          14h
+kube-system          coredns-74ff55c5b-vq827                           1/1     Running     0          14h
+kube-system          etcd-sjc-veba-01.tshirts.inc                      1/1     Running     0          14h
+kube-system          kube-apiserver-sjc-veba-01.tshirts.inc            1/1     Running     0          14h
+kube-system          kube-controller-manager-sjc-veba-01.tshirts.inc   1/1     Running     0          14h
+kube-system          kube-proxy-mwpxs                                  1/1     Running     0          14h
+kube-system          kube-scheduler-sjc-veba-01.tshirts.inc            1/1     Running     0          14h
+local-path-storage   local-path-provisioner-5696dbb894-7x626           1/1     Running     0          14h
+rabbitmq-system      rabbitmq-cluster-operator-7bbbb8d559-dqd85        1/1     Running     0          14h
+vmware-functions     default-broker-ingress-5c98bf68bc-2zpc6           1/1     Running     0          14h
+vmware-functions     sockeye-65697bdfc4-cmfxc                          1/1     Running     0          14h
+vmware-functions     sockeye-trigger-dispatcher-7f4dbd7f78-n589p       1/1     Running     0          14h
+vmware-system        cadvisor-sk4j9                                    1/1     Running     0          14h
+vmware-system        tinywww-dd88dc7db-dqnnc                           1/1     Running     0          14h
+vmware-system        veba-rabbit-server-0                              1/1     Running     0          14h
+vmware-system        veba-ui-54967b4bf4-lpjrn                          1/1     Running     0          14h
+vmware-system        vmware-event-router-vcenter-6b76959df5-6mrb4      1/1     Running     3          14h
+vmware-system        vmware-event-router-webhook-6b48cc5b8c-sjzx8      1/1     Running     0          14h
+```
+
+First, we want to see if the event router is capturing events and forwarding them on to a function.
 
 Use this command to follow the live Event Router log.
 
 ```bash
-kubectl logs -n vmware vmware-event-router-5dd9c8f858-9g44h --follow
+kubectl logs -n vmware-system vmware-event-router-vcenter-6b76959df5-6mrb4 -f
 ```
 
-For this sample troubleshooting, we have the sample hostmaintenance alarms function running. To see if the appliance is properly handling the event, we put a host into maintenance mode. 
+For this sample troubleshooting, we have the sample [PowerCLI Tagging function](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/knative/powercli/kn-pcli-tag) running which will react to a VM powered on Event (`DrsVmPoweredOnEvent`). To see if the appliance is properly handling the event, we create a test VM and power it on.
```

Comment:
```suggestion
For this sample troubleshooting, we have the sample [PowerCLI Tagging function](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/knative/powercli/kn-pcli-tag) running which will react to a VM powered on Event (`DrsVmPoweredOnEvent`). To see if the appliance is properly handling the event, create a test VM and power it on before proceeding with the next steps.
```

Also text says "we have the tagging function running" but it's not deployed (seen in above all pods output). Might be confusing